cafy_pytest/plugin.py

```diff
@@ -672,6 +699,17 @@ def pytest_collection_modifyitems(session, config, items):
         except:
             log.debug("selected_nodeids = ", selected_nodeids)
             log.error("Error while picking selective testcases")
+    else:
+        log.info("No selective testcases file found, therefore executing all testcases in the file")
+
+    if config.option.exhaustive_sort:
+        log.info("Exhaustive sorting of TCs enabled")
+        sorted_items = custom_exhaustive_sort(items)
+        if len(sorted_items) == len(items):
```

Comment:
in which cases this will never be true ? this should sort anyway.
is it for the non-fixture things

Reply:
I haven't found a case where it will not be true but I added it just in case as a roll back to default order if the sorting misbehaves or fails resulting in lesser number or duplication of test cases in the sorted array.